views/pages/index.ejs

```diff
@@ -70,9 +75,9 @@
           <ul class="forecast-secondary">
             <% for (var i = 1; i < 12; i++) { %>
             <li>
-              <span class="data-secondary"><%= moment.unix(hourly.data[i].time).format('hA')%></span>
-              <span class="m-x-1"><%= Math.round(hourly.data[i].temperature) %>º</span>
-              <span><%- include('../partials/summary', {data: hourly.data[i]}) %></span>
+              <span class="data-secondary"><%= moment.unix(hourly.data[i + (dataIndex * 24)].time).format('hA')%></span>
```

Comment:
Doesn't this introduce the bug that we talked about in https://github.com/JulianNorton/weather-10kb/commit/32415e5570ffdf2e80ad3fe0ad147480c8053ac0 ? Might be better to separate out the 'today' and 'tomorrow' summary list entirely.